pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -138,6 +138,7 @@ func (p *cadvisorStatsProvider) ListPodStats(_ context.Context) ([]statsapi.PodS
 				containerStat.Logs = logs
 			}
 			podStats.Containers = append(podStats.Containers, *containerStat)
+			podStats.ProcessStats = mergeProcessStats(podStats.ProcessStats, cadvisorInfoToProcessStats(&cinfo))
```

Comment:
Maybe you could leave a comment here to mention why we are computing processes at the container level.